format-delimited/pom.xml

```diff
@@ -29,6 +29,13 @@
   <name>Delimited format plugins</name>
   <packaging>jar</packaging>
 
+  <properties>
+    <google-cloud-storage.version>1.112.0</google-cloud-storage.version>
```

Comment:
why are all these dependencies added?

Reply:
The format-delimited plugin is responsible to access the specified gs path, investigate one of the blobs and inferr the schema out of it. This can all happen during the configuration time (without the plugin being deployed). For this reason, I've included the `google-cloud-storage` dependency.